src/SearchFilter/SearchFilter.js

```diff
@@ -1,31 +1,35 @@
 //imports
-import React, { Component } from 'react';
-import FilterButton from '../FilterButton/FilterButton.js';
+import React from 'react';
+import FilterButton from '../FilterButton/FilterButton';
+import { connect } from 'react-redux';
+import { fetchAssetsIfNeeded } from '../actions/searchActions';
 
-class SearchFilter extends Component {
+export const SearchFilter = (props) => {
```

Comment:
Please, use the following  way for props in functional components `SearchFilter = ({ value, handleSearchBtnClick })`. It makes code much cleaner.